meta-geico/conf/templates/default/bblayers.conf.sample

```diff
@@ -0,0 +1,16 @@
+# LAYER_CONF_VERSION is increased each time build/conf/bblayers.conf
+# changes incompatibly
+LCONF_VERSION = "8"
+
+BBPATH = "${TOPDIR}"
+BBFILES ?= ""
+
+BBLAYERS ?= " \
+  ##OEROOT##/meta \
+  ##OEROOT##/meta-openembedded/meta-oe \
+  ##OEROOT##/meta-openembedded/meta-networking \
+  ##OEROOT##/meta-openembedded/meta-python \
```

Comment:
I don't recall if this is explicitly required. If there's dependencies for this, it's fine. If not, would remove it

Reply:
These are required for native tools to build